lib/drives/pages/drive_detail_page.dart

```diff
@@ -215,29 +220,31 @@ class _DriveDetailPageState extends State<DriveDetailPage> {
     return Scaffold(
       appBar: AppBar(
         title: Text(S.of(context).pageDriveDetailTitle),
-        actions: <Widget>[
-          IconButton(
-            onPressed: () => Navigator.push(
-              context,
-              MaterialPageRoute(
-                builder: (context) => DriveChatPage(
-                  drive: _drive!,
+        actions: _fullyLoaded && _drive!.approvedRides!.isNotEmpty
```

Comment:
Please always show the chat button. I think you should be able to explore the page even if there is no data yet, so that you can form a mental image of what the page will look like when it's full. I think it would benefit usability if you could click the button, and the "No chats yet" text on the chat page could even explain the relationship between rides and chats.

Reply:
And let's use a completely different "empty" picture for the "no chats/rides" page

Reply:
what kind of picture do you have in mind?

Reply:
![image](https://user-images.githubusercontent.com/69083092/213136176-03d1ba88-751a-47db-a4cb-df412ce856dd.png) ?

Reply:
what do the others think?